modules/apps/accessibility/accessibility-menu-web/src/main/resources/META-INF/resources/js/AccessibilitySetting.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayToggle} from '@clayui/form';
+import {localStorage, setSessionValue} from 'frontend-js-web';
+import PropTypes from 'prop-types';
+import React, {useEffect, useState} from 'react';
+
+import {SETTINGS_STRING_VALUES_MAP, getSettingValue} from './util';
+
+const KEY_EVENT = 'Enter';
+
+const AccessibilitySetting = ({setting}) => {
+	const {className, defaultValue, key, label, sessionClicksValue} = setting;
+
+	const [value, setValue] = useState(
+		SETTINGS_STRING_VALUES_MAP[sessionClicksValue]
```

Comment:
You can get lazy initialization here too instead of creating a `useEffect`.

```jsx
const [value, setValue] = useState(() => getSettingValue(defaultValue, sessionClicksValue, key));
```